S2VX.Game/Editor/ToolState/SelectToolState.cs

```diff
@@ -133,15 +133,15 @@ public class SelectToolState : ToolState {
 
         protected override void Update() {
             Editor.NotesTimeline.TickBarNoteSelections.Clear();
-            var lowerBound = Time.Current - Editor.NotesTimeline.SectionLength * NotesTimeline.SecondsToMS / 2;
-            var upperBound = Time.Current + Editor.NotesTimeline.SectionLength * NotesTimeline.SecondsToMS / 2;
+            var lowerBound = Time.Current - Editor.NotesTimeline.SectionLength * Containers.NotesTimeline.SecondsToMS / 2;
+            var upperBound = Time.Current + Editor.NotesTimeline.SectionLength * Containers.NotesTimeline.SecondsToMS / 2;
             foreach (var noteAndTime in SelectedNoteToTime) {
                 if (lowerBound <= noteAndTime.Value && noteAndTime.Value <= upperBound) {
-                    var relativePosition = (noteAndTime.Value - lowerBound) / (Editor.NotesTimeline.SectionLength * NotesTimeline.SecondsToMS);
+                    var relativePosition = (noteAndTime.Value - lowerBound) / (Editor.NotesTimeline.SectionLength * Containers.NotesTimeline.SecondsToMS);
                     var indication = new RelativeBox {
                         Colour = Color4.LimeGreen.Opacity(0.727f),
-                        Width = NotesTimeline.TimelineNoteWidth + 0.009727f,
-                        Height = NotesTimeline.TimelineNoteHeight + 0.1f,
+                        Width = Containers.NotesTimeline.TimelineNoteWidth + 0.009727f,
+                        Height = Containers.NotesTimeline.TimelineNoteHeight + 0.1f,
```

Comment:
I would like to see a `using S2VX.Game.Editor.Containers;` at the top and `Containers.` removed through the file. Where possible, I have been reducing namespace declarations like this.